docs/openapi/howto/archboard_feedback.md

```diff
@@ -0,0 +1,152 @@
+```
+becomes
+``` json
+ "operationId": "NewGroup_BuildSomethingElse",
+```
+
+## Need one client to have all operations directly on it (no sub-client)
+
+Don't use operation groups in your OperationID (remove them if they exist). For any operation with an `_` character, remove the first part.
+
+``` json
+ "operationId": "MyGroup_BuildSomethingElse",
+```
+becomes
+``` json
+ "operationId": "BuildSomethingElse",
+```
+
+## Need one client/builder with sub-clients
+
+-	Use operation groups, and name them as you want your subclients to be called. Operation group can be create in Swagger by using the `_` character in the OperationID 
+(`OperationGroupName_OperationName`)
+-	For C#, use `single-top-level-client: true`
+
+For instance, this:
+``` json
+ "operationId": "MyGroup_BuildSomethingElse",
+```
+
+Will create a subclient for MyGroup, that will have an operation called BuildSomethingElse.
+In various languages this could looks like (pseudo-code):
+```
+MyServiceClient(endpoint, credentials).MyGroup().BuildSomethingElse() // C#
+
+MyServiceClient(endpoint, credentials).my_group.build_something_else() # Python
+```
+
+## Need two or more service clients (regardless of if some needs subclients or not)
+
+-	Make sure you don't have one Swagger with operations that are designed to be in two different clients.
+  Split Swagger files in multiple swagger files if necessary: clients should correspond to a clear set of Swagger files.
+-	Define tags in `Readme.md` for each client input files. Use the `batch` autorest option to create the two clients:-
+
+```yaml
+batch:
+- tag-client1
+- tag-client2
+```
+
+-	For each individual clients, follow the previous guidance on single clients. You may have options specific to only one tag for one client.
+
+## Rename a model name
+
+Just rename the model name directly in Swagger. Since this name is not use in JSON serialization, there is no need to use specific extensions.
+
+## Rename a model attribute
+
+Since attribute name are used in JSON serialization, we need to declare a mapping between the SDK name and the JSON name. It's achieved with the extensions 
+`x-ms-client-name` :
+
+``` json
+    "ModelNAme": {
+      "description": "This model is very helpful.",
+      "properties": {
+        "attributeName": {
+          "x-ms-client-name": "betterSDKNameForThisAttribute",
+          "type": "string"
+        }
+      }
+    }
+```
+
+## Operation should be pageable
+
+You will get this feedback if your operation is returning a list of objects. Contrary to a common misconception: you do not need to support paging on the server to design an operation as a pageable. Pageable is a design, not a server capability.
+
+The full documentation for pageable is available here: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-pageable
+
+At a glance:
+- Add `x-ms-pageable` node to your operation definition
+- Make sure your return schema is an array of T (T could be anything).
```

Comment:
I think `nextLink` has to be added to the return schema -- even if not currently sent by the service -- and documented as a link to the next page of results.  And then `nextLinkName` should be `nextLink`.

Reply:
@mikekistler autorest do not requires it, are you saying that stewardship recommendation is to fake a `nextLink` in the return schema even if that doesn't exist?

Reply:
Changing a REST API from non-pageable to pageable is a breaking change.  If the REST API does not define a `nextLink` property, I don't know how it could be considered "pageable".  So we do recommend defining the `nextLink` property in the response even if the service currently sends all results in a single page.

Reply:
I think it more for the user that directly use the REST API (less about SDK).

If "nextLink" property is not in version1, user will not write code to follow to the next page (which is different if "nextLink" property is there but service always return null in version1).
Then if swagger upgrade to version2 with "nextLink" property, user's code will not follow this new property. User had to modify their code when upgrading api-version.